### PR TITLE
Changes according to common review

### PR DESCRIPTION
**Pull request by rohit-vishwakarma**
[Deployed link ](https://neetoui-rohit-vishwakarma.herokuapp.com/)
email: oliver@example.com
password: welcome

### CODE REVIEW

File: app/javascript/src/utils/index.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
 import * as R from "ramda";
 
+const toCamelCase = str =>
+  str
+    .toLowerCase()
+    .replace(/[^a-zA-Z0-9]+(.)/g, (match, chr) => chr.toUpperCase());
```
Comment (yedhink) [inline app/javascript/src/utils/index.js:6]: So if I provide `UX-Designer` as the input then what would be the output from this util function?

My expectation is `uxDesigner`

File: app/javascript/src/utils/index.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+
+export const buildSelectOptions = stringArrayOptions =>
```
Comment (yedhink) [inline app/javascript/src/utils/index.js:8]: https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663250339031239?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Find a better name.

**Review by yedhink — COMMENTED** [review]
@rohit-vishwakarma

File: app/javascript/src/utils/index.js (hunk 1 above)
Comment (rohit-vishwakarma) [reply app/javascript/src/utils/index.js:6]: @yedhink yes

Comment (rohit-vishwakarma) [reply app/javascript/src/utils/index.js:6]: @yedhink if the input is `UX-Designer`. The output will be `uxDesigner`.

**Comment by yedhink** [comment]
@rohit-vishwakarma I have provided review for all the other PRs in your repo. I haven't repeatedly pointed out each and every mistake that you have made. It's your responsibility to go through the review comments and figure out where all changes ought to be made.

Once changes have been addressed lemme know.

**Pull request merged**